src/Codeception/Module/Asserts.php

```diff
@@ -263,35 +262,6 @@ public function assertFileNotExists($filename, $message = '')
     }
 
     /**
-     * @param $expected
```

Comment:
I don't like this change - you removed methods without warning and even without mentioning that in the changelog, but you left aliases that were marked as deprecated long time ago (assertGreaterThen and assertGreaterThenOrEqual ). This is a BC break if anyone is using them.